App crashes upon startup before accepting permissions

With the recent update to the quickstart app, the following error occurs for first time installs:

FATAL EXCEPTION: main
                                                                             Process: com.twilio.video.quickstart, PID: 12481
                                                                             java.lang.RuntimeException: Unable to resume activity {com.twilio.video.quickstart/com.twilio.video.quickstart.activity.VideoActivity}: java.lang.NullPointerException: VideoCapturer must not be null
                                                                                 at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3103)
                                                                                 at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3134)
                                                                                 at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2481)
                                                                                 at android.app.ActivityThread.-wrap11(ActivityThread.java)
                                                                                 at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344)
                                                                                 at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                                 at android.os.Looper.loop(Looper.java:148)
                                                                                 at android.app.ActivityThread.main(ActivityThread.java:5417)
                                                                                 at java.lang.reflect.Method.invoke(Native Method)
                                                                                 at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
                                                                                 at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
                                                                              Caused by: java.lang.NullPointerException: VideoCapturer must not be null
                                                                                 at com.twilio.video.Preconditions.checkNotNull(Preconditions.java:794)
                                                                                 at com.twilio.video.LocalVideoTrack.create(LocalVideoTrack.java:69)
                                                                                 at com.twilio.video.LocalVideoTrack.create(LocalVideoTrack.java:44)
                                                                                 at com.twilio.video.quickstart.activity.VideoActivity.onResume(VideoActivity.java:161)
                                                                                 at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1258)
                                                                                 at android.app.Activity.performResume(Activity.java:6327)
                                                                                 at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3092)
                                                                                 at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3134) 
                                                                                 at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2481) 
                                                                                 at android.app.ActivityThread.-wrap11(ActivityThread.java) 
                                                                                 at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344) 
                                                                                 at android.os.Handler.dispatchMessage(Handler.java:102) 
                                                                                 at android.os.Looper.loop(Looper.java:148) 
                                                                                 at android.app.ActivityThread.main(ActivityThread.java:5417) 
                                                                                 at java.lang.reflect.Method.invoke(Native Method) 
                                                                                 at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
                                                                                 at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 


What would be the "best practice" way to fix this?  Thanks.

--- a/build.gradle
+++ b/build.gradle
@@ -5,7 +5,7 @@ buildscript {
        jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.0'
+        classpath 'com.android.tools.build:gradle:2.3.1'
     }
 }
 

--- a/quickstart/src/main/java/com/twilio/video/quickstart/activity/VideoActivity.java
+++ b/quickstart/src/main/java/com/twilio/video/quickstart/activity/VideoActivity.java
@@ -157,7 +157,7 @@ public class VideoActivity extends AppCompatActivity {
         /*
          * If the local video track was released when the app was put in the background, recreate.
          */
-        if (localVideoTrack == null) {
+        if (localVideoTrack == null && checkPermissionForCameraAndMicrophone()) {
             localVideoTrack = LocalVideoTrack.create(this, true, cameraCapturer);
             localVideoTrack.addRenderer(localVideoView);
 

